mmv1/templates/terraform/examples/vertex_ai_featureonlinestore_featureview_cross_project.tf.erb

```diff
@@ -0,0 +1,132 @@
+resource "google_project" "project" {
+  project_id      = "tf-test%{random_suffix}"
+  name            = "tf-test%{random_suffix}"
+  org_id          = "<%= ctx[:test_env_vars]['org_id'] %>"
+  billing_account = "<%= ctx[:test_env_vars]['billing_account'] %>"
+}
+
+resource "google_project_service" "bigquery" {
+  service = "bigquery.googleapis.com"
+  project = google_project.project.project_id
+  timeouts {
+    create = "30m"
+    update = "40m"
+  }
+  disable_on_destroy = false
+}
+
+resource "google_project_service" "vertexai" {
+  service = "aiplatform.googleapis.com"
+  project = google_project.project.project_id
+  timeouts {
+    create = "30m"
+    update = "40m"
+  }
+  disable_on_destroy = false
+}
+
+resource "google_vertex_ai_feature_online_store" "featureonlinestore" {
+  name     = "<%= ctx[:vars]['name'] %>"
```

Comment:
Should this be part of `google_project.project`? If not, should we remove the dependency on the `google_project_service` resource?

Reply:
you're right, thanks for pointing it out. I removed the dependency now on featureonlinestore.

Reply:
I've tried to replicate this by creating a new serviceAccount with only owner role and impersonated it to create FeatureGroups. It works fine.